installing/installing_platform_agnostic/installing-platform-agnostic.adoc

```diff
@@ -0,0 +1,136 @@
+[id="installing-platform-agnostic"]
+= Installing a cluster on any platform
+include::modules/common-attributes.adoc[]
+:context: installing-platform-agnostic
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+any infrastructure that you provision, including virtualization and cloud environments.
+
+[IMPORTANT]
+====
+ Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]
+before you attempt to install an {product-title} cluster in
+virtualized or cloud environments.
+====
+
+== Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-infrastructure-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-network-user-infra.adoc[leveloffset=+2]
+
+include::modules/installation-dns-user-infra.adoc[leveloffset=+2]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+
+include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+include::modules/installation-three-node-cluster.adoc[leveloffset=+1]
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
+
+[id="creating-machines-platform-agnostic"]
+== Creating {op-system-first} machines
+
+Before you install a cluster on infrastructure that you provision, you must create {op-system} machines for it to use. If your infrastructure supports it, create the machines by following either the steps to use an ISO image or network PXE booting. In some cases, you might be able to upload an appropriate {op-system} image from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat customer portal or the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror] to your cloud provider and use that image to create the machines.
```

Comment:
Do we treat the Red Hat Customer Portal as a proper noun?

Reply:
In most places. I'll fix the rest.